README_en.md

```diff
@@ -0,0 +1,32 @@
+# What is this?
+This repository is Python implementation program of SOM(Self-Organaizing Map)and its derived algorithms that we open to publish
+
+This repository is operated by Furukawa Laboratory of Kyushu Institute of Technology. Our laboratory has developed a machine learning algorithm based on SOM that is useful for knowledge discovery from data.In this repository, not only the basic SOM, but also the implementation of the algorithm that developed in this laboratory.
+
+This repository was originally maintained as private in order to share the algorithm in our laboratory, but this time it has been made public. We are currently working on the release as a library. Of course, contributions from outsiders are very welcome.
+
+# What is SOM?
+SOM is a type of neural network, and is a widely used algorithm for visualization and modeling of high dimensional data. It has existed since ancient times, there are aspects that are also related to algorithms for machine learning and deep learning that show upsurge recently. Please refer to the [document](http://www.brain.kyutech.ac.jp/~furukawa/data/SOMtext.pdf) published by our laboratory for details.
```

Comment:
```suggestion
SOM is a type of neural network, and is a widely used algorithm for visualization and modeling of high dimensional data. It has existed since ancient times, there are aspects that are also related to algorithms for machine learning and deep learning that show upsurge recently. Please refer to the [document](http://www.brain.kyutech.ac.jp/~furukawa/data/SOMtext.pdf) published by our laboratory for details (Japanese only).
```

Reply:
日本語だけだよーって後ろに書いといた！